js/ui/codeView.js

```diff
@@ -1465,6 +1466,15 @@ var CodeViewManager = GObject.registerClass({
         if (!global.settings.get_boolean('hack-mode-enabled'))
             return false;
 
+        if (!Clubhouse.getClubhouseApp()) {
+            // If the clubhouse app is not installed and the hack-mode-enabled
+            // is true we've a problem because the clubhouse was uninstalled
```

Comment:
It could be as well that a user has set the setting (I know not too likely). I would phrase it in that way.